apollo-spaceport/.gitignore

```diff
@@ -0,0 +1 @@
+/target
```

Comment:
A subdirectory's gitignore in case the repository's gitignore doesn't work? :grin:

Reply:
Good catch. Leftover from when this code was in a separate repo. Removed.
done: 6ff86de619bd6268ff41db6fce9da5f3d3dc22ea